src/frontend/magnify/packages/components/src/components/jitsi/MagnifyMeeting/index.tsx

```diff
@@ -71,6 +72,7 @@ export const MagnifyMeeting = ({
   const getConfig = (): object => {
     return {
       ...configuration,
+      subject: props.roomDisplayName,
```

Comment:
you don't set localSubject anymore ?

Reply:
You had to do the review before my change but I had already removed it because it was useless.